Devscord.DiscordFramework/Services/MessagesService.cs

```diff
@@ -63,6 +64,11 @@ public async Task SendFile(string filePath)
             var channel = (IRestMessageChannel) await Server.GetChannel(this.ChannelId);
             await channel.SendFileAsync(filePath);
         }
+        public async Task SendFile(Stream stream, string fileName)
+        {
+            var channel = (IRestMessageChannel)await Server.GetChannel(this.ChannelId);
+            await channel.SendFileAsync(stream, fileName + ".png");
```

Comment:
dlaczego na sztywno `.png` i co jeśli tam już jest .png?

Reply:
W złym miejscu dodałem .png, zwróciłeś na to uwagę w innym komentarzu, jak najbardziej zmieniam